src/FixedDataTableRow.js

```diff
@@ -219,6 +220,7 @@ class FixedDataTableRowImpl extends React.Component {
         width={fixedRightColumnsWidth}
         zIndex={2}
         columns={this.props.fixedRightColumns}
+        columnOffsets={this.props.fixedRightColumnOffsets}
```

Comment:
Are this and fixedColumnOffsets on line 199 meant to be part of this review? I'm not seeing them used anywhere on this branch except here.

I'm also confused why I'm not seeing columnGroupOffsets or columnGroupsToRender being used anywhere after being passed into FDTRow from FDT. Is this covered in other reviews?

Reply:
Looks like they're used in #462 and the changes to Cell / CellGroup rendering can be found there.